PolyEngine/RenderingDevice/OpenGL/Src/GLShaderProgram.hpp

```diff
@@ -66,5 +66,9 @@ namespace Poly {
 		std::map<String, OutputInfo> Outputs;
 		GLuint ProgramHandle;
 		EnumArray<String, eShaderUnitType> ShaderCode;
+
+		String VertexProgramPath;
+		String GeometryProgramPath;
+		String FragmentProgramPath;
```

Comment:
As stated in previous review, this fields should be removed.

Reply:
done